app/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp:3.12.2'
     implementation 'com.squareup.okhttp3:logging-interceptor:3.12.2'
 
-    implementation project(path: ':locationsyncservice')
```

Comment:
let's have it locally. We might be changing flows in the sdk and test it in to app module

Reply:
@nikhil-roy updated

Reply:
thanks